tests/dialects/test_dialects.py

```diff
@@ -0,0 +1,18 @@
+from xdsl.dialects import get_all_dialects
+
+
+def test_op_names():
```

Comment:
```suggestion
def test_op_class_names():
```
Just to make it obvious we're not checking the `name` field of the operation?